packages/cli/src/util/createServer.ts

```diff
@@ -22,23 +22,32 @@ export async function createMultiProcessPrism(options: CreatePrismOptions) {
     }
   } else {
     const logInstance = createLogger('CLI');
-    return createPrismServerWithLogger(options, logInstance);
+    try {
+      return await createPrismServerWithLogger(options, logInstance);
+    } catch (e) {
+      logInstance.fatal(e.message);
+      cluster.worker.kill();
+    }
   }
 }
 
-export function createSingleProcessPrism(options: CreatePrismOptions) {
+export async function createSingleProcessPrism(options: CreatePrismOptions) {
```

Comment:
No async if there is no await 
```suggestion
export function createSingleProcessPrism(options: CreatePrismOptions) {
```